Source/WebCore/html/HTMLDialogElement.cpp

```diff
@@ -132,18 +136,20 @@ void HTMLDialogElement::queueCancelTask()
 }
 
 // https://html.spec.whatwg.org/multipage/interactive-elements.html#dialog-focusing-steps
-void HTMLDialogElement::runFocusingSteps()
+void HTMLDialogElement::runFocusingSteps(bool isModal)
```

Comment:
I don't think we need to replace `m_isModal` with this parameter. You can't turn a modal dialog into a popover (`checkPopoverValidity` prevents it), and you can't run `showModal()` on open popovers either.

Reply:
Done.